modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1101,6 +1127,34 @@ private Map<String, ObjectRelationship> _getObjectRelationships(
 		return objectRelationships;
 	}
 
+	private long _getRelatedModelPrimaryKey(
```

Comment:
The purpose of this method was to parse the Object passed by argument and return the primaryKey, not calling the fetch methods. You are duplicating code with the casting code (see `.getPrimaryKeyObj();`) and this method is called only once